.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
\ No newline at end of file
+package-lock=true
```

Comment:
I would just remove the file since this is the default.

Reply:
Also, this seems to be a v2 lock file so I think the Dockerfile and .drone.yml need to be updated to a version that ships with npm 8.

Reply:
File removed, need to look into the drone config still.